docs/bloqs/index.rst

```diff
@@ -29,6 +29,7 @@ Bloqs Library
     basic_gates/toffoli.ipynb
     basic_gates/hadamard.ipynb
     basic_gates/cnot.ipynb
+    basic_gates/su2_rotation.ipynb
```

Comment:
@mpharrigan  was adding `su2_rotation` to the `Rotations:` section in https://github.com/quantumlib/Qualtran/pull/843

Reply:
ah yes, will rebase on top of that